src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -58952,13 +58983,14 @@ public void F2<TF2A>()
     {}
 }
 ";
-            var comp = CreateCompilation(new[] { source }, options: WithNonNullTypesTrue());
-            // https://github.com/dotnet/roslyn/issues/29980: unexpected warning
-            comp.VerifyDiagnostics(
-                // (23,17): warning CS8633: Nullability in constraints for type parameter 'TF2A' of method 'A.F2<TF2A>()' doesn't match the constraints for type parameter 'TF2' of interface method 'I1.F2<TF2>()'. Consider using an explicit interface implementation instead.
-                //     public void F2<TF2A>()
-                Diagnostic(ErrorCode.WRN_NullabilityMismatchInConstraintsOnImplicitImplementation, "F2").WithArguments("TF2A", "A.F2<TF2A>()", "TF2", "I1.F2<TF2>()").WithLocation(23, 17)
-                );
```

Comment:
The LDM specified that a disabled region suppresses nullability warnings.  The bug here was that we were producing a nullability warning in a disabled region.  It doesn't appear you fixed it by adjusting where we suppress or produce nullability diagnostics.

Reply:
> The LDM specified that a disabled region suppresses nullability warnings.  The bug here was that we were producing a nullability warning in a disabled region.  It doesn't appear you fixed it by adjusting where we suppress or produce nullability diagnostics.

The warnings can be enabled even when declarations are disabled. I'll adjust the test to reflect that. So, this isn't an issue about suppression isn't working. We do have an issue #32742 tracking the work about finalizing what warning should be suppressed when nullable warnings are disabled.

---
In reply to: [272739261](https://github.com/dotnet/roslyn/pull/34797#discussion_r272739261) [](ancestors = 272739261)

Reply:
Please add a comment.

---
In reply to: [272763113](https://github.com/dotnet/roslyn/pull/34797#discussion_r272763113) [](ancestors = 272763113,272739261)

Reply:
> Please add a comment.

I do not think any comment is warranted here. As I mentioned in the previous reply to this thread, I am planning to adjust the test to enable nullability warnings explicitly

---
In reply to: [272766744](https://github.com/dotnet/roslyn/pull/34797#discussion_r272766744) [](ancestors = 272766744,272763113,272739261)

Reply:
OK.  Reopening thread to track that a change (enable nullability warnings explicitly) is pending for this test.

---
In reply to: [272768576](https://github.com/dotnet/roslyn/pull/34797#discussion_r272768576) [](ancestors = 272768576,272766744,272763113,272739261)

Reply:
> Reopening thread to track that a change (enable nullability warnings explicitly) is pending for this test.

This not pending, it is done

---
In reply to: [273195151](https://github.com/dotnet/roslyn/pull/34797#discussion_r273195151) [](ancestors = 273195151,272768576,272766744,272763113,272739261)